benchmark/test.csv

```diff
@@ -0,0 +1,33 @@
+n,n,1,2,3
```

Comment:
please move this to a folder with ranges. e.g. `benchmark/configurations/test.csv` or `benchmark/ranges/test.csv`

Reply:
Actually I added this file by mistake, I will remove it

Reply:
Done